example/src/KlaviyoReactWrapper.ts

```diff
@@ -50,7 +52,7 @@ export const setExternalId = async () => {
 export const getEmail = async () => {
   try {
     Klaviyo.getEmail((value: string) => {
-      console.log('email is =', value);
+      console.log('email = ', value);
```

Comment:
Is there a way to do a toast in rn?

Reply:
It'd be a nice addition yeah, I think in first pass we were keeping this sample app light on dependencies, but I imagine there's a cross-platform toast lib. iOS doesn't have toast exactly, but you can use alerts in a less intrusive way.